HealthcareApp/src/Healthcare.Application/DependencyInjection.cs

```diff
@@ -0,0 +1,16 @@
+using Application.DTOs.Register;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Healthcare.Application;
+
+public static class DependencyInjection
+{
+    public static IServiceCollection AddApplication(this IServiceCollection services)
+    {
+        var assembly = typeof(RegisterUserDTOValidator).Assembly; 
```

Comment:
Use empty `ApplicationAssemblyReference` class for this use case also. This is what we've discussed in tg